locale/zh_CN/manageiq.po

```diff
@@ -25389,6 +25395,9 @@ msgstr "无效验证证书"
 msgid "Invalid button action"
 msgstr "无效的按钮操作"
 
+msgid "Invalid button action!"
+msgstr "按钮操作无效！"
```

Comment:
ugh, space after !

Reply:
lol, https://www.codetable.net/decimal/65281

That's a unicode ! with a deceptively large amount of visual space after it.  Keeping it even if my 👀 don't like the extra space.